Gems/Atom/RHI/Code/Include/Atom/RHI/MultiDeviceImage.h

```diff
@@ -141,10 +143,22 @@ namespace AZ::RHI
             return m_descriptor;
         }
 
+        const MultiDeviceResource* GetResource() const override
+        {
+            return m_image;
+        }
+
+        const ResourceView* GetDeviceResourceView(int deviceIndex) const override
+        {
+            return GetDeviceImageView(deviceIndex).get();
+        }
+
     private:
         //! A raw pointer to a multi-device image
         const RHI::MultiDeviceImage* m_image;
         //! The corresponding ImageViewDescriptor for this view.
         ImageViewDescriptor m_descriptor;
+        //! ImageView cache
```

Comment:
Add a comment why we are caching this here and if are planning on better alternative in future.